src/modules/plan_indicativo_auditoria/plan_indicativo_auditoria.service.ts

```
@@ -0,0 +1,643 @@
+                ? (mp.enfoque_poblacional_ids as any[]).map((v: any) => Number(v))
+                : [];
+            const idsFromDb = enfoquesByMetaId.get(Number(mp.id)) || [];
+            const selectedIds = new Set<number>([...idsFromSnapshot, ...idsFromDb].filter((v) => Number.isFinite(v)));
+            if (selectedIds.size > 0) {
+                this.enfoquePoblacionalColumns.forEach(({ col, id }) => {
+                    if (selectedIds.has(id)) worksheet.getCell(`${col}${currentRow}`).value = 'X';
+                });
+            }
+
+            // Enfoque Territorial (Sector): 1=Urbano, 2=Rural
+            const enfoqueTerritorial = Array.isArray((mp as any)?.enfoque_territorial) ? (mp as any).enfoque_territorial : [];
```

ESLint

[error] Replace `············const·enfoqueTerritorial·=·Array.isArray((mp·as·any)?.enfoque_territorial)·?·(mp·as·any).enfoque_territorial·:·[];` with `······const·enfoqueTerritorial·=·Array.isArray((mp·as·any)?.enfoque_territorial)⏎········?·(mp·as·any).enfoque_territorial`

(prettier/prettier)